src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,83 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_STORAGE_SIZE = 10;
+    private final Object[] keyArr = new Object[MAX_STORAGE_SIZE];
+    private final Object[] valuesArr = new Object[MAX_STORAGE_SIZE];
+    private int currentSize = 0;
```

Comment:
int has default value 0
```suggestion
    private int size;
```